### PR TITLE
Enable `flutter drive` on Windows

### PR DESCRIPTION
**Pull request by goderbauer**
... and enable some more tests on Windows.

### CODE REVIEW

File: packages/flutter_tools/test/drive_test.dart [hunk 1]
```diff
@@ -297,10 +302,18 @@ void main() {
...
         expect(device.name, 'mock-android-device');
       }, overrides: <Type, Generator>{
-        FileSystem: () => memoryFileSystem,
+        FileSystem: () => fs,
       });
+    }
+
+    group('findTargetDevice on Linux', () {
+      findTargetDeviceOnLinuxOrWindows(isLinux: true);
+    });
```
Comment (yjbanov) [inline packages/flutter_tools/test/drive_test.dart]: Should this test be skipped on Windows? Likewise, should the test below be skipped on Linux?

**Review by yjbanov — APPROVED** [review]
LGTM

File: packages/flutter_tools/test/drive_test.dart (hunk 1 above)
Comment (goderbauer) [inline packages/flutter_tools/test/drive_test.dart]: The actual platform is mocked out for these tests. That way, we can test the Windows code path on Linux and vice versa.

Comment (yjbanov) [inline packages/flutter_tools/test/drive_test.dart]: Nice!

**Pull request merged**